Github/service.js

```diff
@@ -228,3 +228,73 @@ module.exports.getIssues = repos => {
       });
   });
 };
+
+const getPullRequest = repo => {
+  let pullRequestData = {};
+  const pullRequestsPerBranch = [];
+  return new Promise((resolve, reject) => {
+    request.get(
+      {
+        url: `/repos/Seneca-CDOT/${repo}/pulls`,
+      },
+      (err, res, data) => {
+        if (res.statusCode !== 200) {
+          console.log('Error:', res.statusMessage);
+          reject(new Error('Unable to get pull requests.'));
+        } else {
+          JSON.parse(data).forEach(pullRequest => {
+            pullRequestData = {
+              title: pullRequest.title,
+              author: {
+                name: pullRequest.user.login,
+                avatar: pullRequest.user.avatar_url,
+              },
+              created: new Date(pullRequest.created_at).toLocaleString('en-US', {
+                timeZone: 'America/Toronto',
+              }),
+              reviewers: pullRequest.requested_reviewers.map(reviewer => {
+                return {
+                  name: reviewer.login,
+                  avatar: reviewer.avatar_url,
+                }
+              }),
+              description: pullRequest.body,
+              repoName: repo,
+              url: pullRequest.html_url,
+              labels: pullRequest.labels.map(label => {
+                return {
+                  name : label.name,
+                  color : label.color
+                }
+              }),
+              number: pullRequest.number,
+            };
+            pullRequestsPerBranch.push(pullRequestData);
+          });
+          resolve(pullRequestsPerBranch);
+        }
+      },
+    );
+  });
+};
+
+module.exports.getPullRequests = repos => {
+  return new Promise((resolve, reject) => {
+    const promises = [];
+    repos.forEach(repo => {
+      promises.push(getPullRequest(repo.name));
```

Comment:
Function names are a bit confusing, you got `getPullRequests` and `getPullRequest` where the latter may return multiple pull requests despite promising a single thing.